sd-svs/decrypt-sd-submission

```diff
@@ -137,11 +137,5 @@ fh.close()
 
 shutil.rmtree(tmpdir)
```

Comment:
do we need the lines above starting in `fh = tempfile.NamedTemporaryFile(suffix=".sd-xfer-extracted", delete=False)`? looks like no since we have a directory with the decrypted files in a folder in `/tmp` which we can just directly move into place in `~/Sources` (previously we needed this logic as we had to ship off a directory of files/messages to another VM)

Reply:
great points @redshiftzero . I have added a commit to address the issue raised and updated the diagrams,